src/components/alert/alert.tsx

```diff
@@ -128,6 +128,7 @@ export class Alert {
 
   disconnectedCallback(): void {
     window.clearTimeout(this.autoDismissTimeoutId);
+    this.containerDiv.removeEventListener("transitionrun", this.onTransitionRun);
```

Comment:
Maybe check to see if containerDiv is defined first. I think there's a chance it might get disconnected before first render.